stdlib/set.mli

```diff
@@ -118,6 +118,16 @@ module type S =
        The elements of [s] are presented to [f] in increasing order
        with respect to the ordering over the type of the elements. *)
 
+    val map: (elt -> elt) -> t -> t
+    (** If the elements of [s] are [a0],[a1],..[aN], then the elements
+        of [map f s] are [f a0],[f a1]... [f aN].
```

Comment:
I personally prefer the documentation pattern "[map f s] is ..." it directly tells you how to read the occurence of such an expression in a program.